locales/en/LC_MESSAGES/django.po

```diff
@@ -19,20 +19,15 @@ msgstr ""
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
 
 #: frontend/templates/frontend/safe_mode_ui.html:9
-msgid ""
-"This site is currently in <strong>compatibility mode</strong>. For an "
-"enhanced experience, you can disable it, but this may cause compatibility "
-"issues with your current browser."
+msgid "This site is currently in <strong>compatibility mode</strong>. For an enhanced experience, you can disable it, but this may cause compatibility issues with your current browser."
```

Comment:
Nothing actually changed here, it's just a result of using `--no-wrap` w/ Django's `makemessages` command... it doesn't do much this file (our English translations) but it helps the Spanish ones not be in a constant state of churn, where Crowdin writes them out without line wrapping in one PR and Django wraps them in another PR, making diffs more confusing (see ee71e2ab5b0ea0038ab563c8bc58fb143a26b342 for an example of the confusing behavior).

Reply:
Ahhh yeah I could imagine that getting frustrating!